docs/develop/convert-javascript-to-typescript.md

```diff
@@ -43,6 +32,41 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 ## Convert the add-in project to TypeScript
 
+### Add Nuget packages
+
+1. Open the Nuget package manager by choosing **Tools > Nuget Package Manager > Manage Nuget Packages for Solution**
+1. Select the **Browse** tab and search for **Microsoft.TypeScript.MSBuild**. Install this package to the ASP.NET web project, or update it if it's already installed. This will ensure the project will transpile to JavaScript when the build runs.
+1. On the **Browse** tab, search for  **jquery.TypeScript.DefinitelyTyped**. Install this package to the ASP.NET web project, or update it if it's already installed. This will ensure the jQuery TypeScript definitions are included in your project. The packages for jQuery appear in a file generated by Visual Studio, called **packages.config**.
```

Comment:
Is either project an ASP.NET web project? there's nothing explicitly labelled that way. Maybe just say "Install this package to the whole solution."

Reply:
Made a change so hopefully this is clearer.